src/core/bridge.ts

```diff
@@ -19,6 +27,23 @@ export interface BridgeCallback {
   (err?: Error, res?: Results): void;
 }
 
+export class BridgeTimeout extends Error {
+  constructor (err: string) {
+    super(err);
```

Comment:
we lost overall coverage because of these lines, can we add a simple test to this